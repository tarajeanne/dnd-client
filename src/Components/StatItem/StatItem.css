.midcol {
  text-align: center;
}

tr:nth-child(odd) {
  background-color: white;
}
