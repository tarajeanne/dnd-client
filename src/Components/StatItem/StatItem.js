import React from 'react';
import './StatItem.css'

class StatItem extends React.Component {
  render() {
    return (
      <tr>
        <td>{this.props.skill}</td>
        <td className="midcol">{this.props.mod}</td>
        <td className="midcol">{this.props.prof}</td>
        <td className="midcol">{this.props.total}</td>
      </tr>
    );
  }
}

export default StatItem;