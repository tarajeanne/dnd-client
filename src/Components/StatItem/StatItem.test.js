import React from 'react';
import StatItem from './StatItem';
import {shallow} from 'enzyme';
import toJson from 'enzyme-to-json';

describe('StatItem', () => {
    it('renders as expected', () => {
        const wrapper = shallow(<StatItem />);
        expect(toJson(wrapper)).toMatchSnapshot();
    })
  });
