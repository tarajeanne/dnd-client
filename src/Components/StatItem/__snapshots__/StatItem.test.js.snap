// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StatItem renders as expected 1`] = `
<tr>
  <td />
  <td
    className="midcol"
  />
  <td
    className="midcol"
  />
  <td
    className="midcol"
  />
</tr>
`;
